crates/fiber-lib/src/fiber/network.rs

```diff
@@ -851,11 +858,22 @@ where
 
     pub async fn handle_peer_message(
         &self,
+        myself: ActorRef<NetworkActorMessage>,
         state: &mut NetworkActorState<S>,
         peer_id: PeerId,
         message: FiberMessage,
     ) -> crate::Result<()> {
         match message {
+            FiberMessage::Init(init_message) => {
```

Comment:
If the remote node hasn't sent us an init message, it looks like the current PR will still handle other messages from the remote node ? (e.g., OpenChannel, etc.).

Reply:
fixed https://github.com/nervosnetwork/fiber/pull/719/commits/4c8db436c3d152184e7dd546cf742a31884d0a8d